### ISSUE TITLE
missing instruction for SKR1.3 and max 31865/ mintemp error

### ISSUE THREAD

**Issue by pietja5** [bigtreetech/BIGTREETECH-MAX31865#2]
MAX31865 is not working with SKR1.3 
issue created on Marlin repository 
https://github.com/GadgetAngel/MAX31865-SKR-V1-3-GUIDE - is not working

**Comment by MrMitch17** [comment]
Yes, Please include Instructions on how to integrate with SKR 1.3.  Thank you.
